cartridge/shop/templates/admin/shop/product/change_list.html

```diff
@@ -0,0 +1,26 @@
+{% extends "admin/change_list.html" %}
+{% load shop_tags pages_tags i18n staticfiles %}
+
+{% block extrahead %}
+{{ block.super }}
+
+<script src="{% static "mezzanine/js/jquery-ui-1.8.14.custom.min.js" %}"></script>
+<script src="{% static "mezzanine/js/admin/page_tree.js" %}"></script>
+{% endblock %}
```

Comment:
Works fine without them so I removed. I wasn't 100% sure what page_tree.js does but it doesn't look relevant.